templates/base.html

```diff
@@ -37,7 +38,25 @@
     </div>
   </div>
 </header>
+<script>
+document.addEventListener("DOMContentLoaded", function () {
+  function attachLinkListeners() {
```

Comment:
Could you please explain this method in a comment, so I understand what is its purpose ?

Reply:
This loads the site with an request and replaces the current site. So in webapp mode it won't show the navigation. 
Maybe this doesn't work when using more JavaScript on the page. 
Maybe there is an better method to build a webapp. 
But this works so far.